### PR TITLE
RFR: Change arrow color when user upvotes/downvotes

### PR DESCRIPTION
**Pull request by Tobitron**
We added user_upvoted and user_downvoted methods in our review model to determine if a user has voted. Not the prettiest methods, but we've spent a ton of time on this feature and in the interest of getting to other features before Friday wanted to push this and refactor later if possible!

### CODE REVIEW

**Comment by dtnorris** [comment]
I'm happy for you guys to be done with this and move on. But you need tests, that is one shortcut you can't take.

File: app/models/review.rb [hunk 1]
```diff
@@ -28,4 +28,22 @@ def sum_of_votes
     end
     sum
   end
+
+  def user_upvoted?(user)
```
Comment (dtnorris) [inline app/models/review.rb:32]: Also just as a stylistic thing I tend to only use `unless` if I'm doing an inline operation like:

```
return unless flowdock.open?
```

if you are nesting conditional logic like below I'd use the following syntax:

```
if !(user.nil?)
    vote = votes.find_by(user_id: user.id)
    if !(vote.nil?)
        vote.value == 1
    end
end
```

**Comment by dtnorris** [comment]
I want to accept this PR guys, can you include a test for `user_downvoted?` as well please?

**Comment by Tobitron** [comment]
Added a unit test for the user_downvoted? method, sorry about that, thanks for keeping on us!

**Pull request merged**